### PR TITLE
Python 3 compatibility. Update RAI stations list. Unified bbc_radio.py and rai_radio.py in a single file

### PR DESCRIPTION
**Pull request by jacopomoioli**
I tried to update this script. it's my first pull request so excuse me if I did something wrong ;-)

### CODE REVIEW

File: radio.py [hunk 1]
```diff
@@ -67,41 +68,67 @@
...
+    ['RAI Radio 1 Sport', 'http://icestreaming.rai.it/13.mp3'],
+    ['RAI Radio 2', 'http://icestreaming.rai.it/2.mp3'],
+    ['RAI Radio 2 Indie', 'http://icestreaming.rai.it/15.mp3'],
+    ['RAI Radio 3', 'http://icestreaming.rai.it/3.mp3'],
+    ['RAI Radio 3 Classica', 'http://icestreaming.rai.it/5.mp3'],
+    ['RAI IsoRadio', 'http://icestreaming.rai.it/6.mp3'],
+    ['RAI GrParlamento', 'http://icestreaming.rai.it/7.mp3'],
+    ['RAI Radio Kids', 'http://icestreaming.rai.it/11.mp3'],
+    ['RAI Radio Live', 'http://icestreaming.rai.it/10.mp3'],
+    ['RAI Radio Techete\'', 'http://icestreaming.rai.it/9.mp3'],
```
Comment (stefantalpalaru) [inline radio.py]: This should be "Techetè". What you're using is an alternative spelling trying to simulate accents when only ASCII is available, but we can use proper Unicode here.

File: radio.py [hunk 2]
```diff
@@ -9,23 +9,24 @@
...
+    ['BBC Radio 1Xtra', 'http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/http-icy-mp3-a/vpid/bbc_1xtra/format/pls.pls'],
+    ['BBC Radio 2', 'http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/http-icy-mp3-a/vpid/bbc_radio_two/format/pls.pls'],
+    ['BBC Radio 3', 'http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/http-icy-mp3-a/vpid/bbc_radio_three/format/pls.pls'],
+    ['BBC Radio 4', 'http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/http-icy-mp3-a/vpid/bbc_radio_fourfm/format/pls.pls'],
+    ['BBC Radio 4Xtra', 'http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/http-icy-mp3-a/vpid/bbc_radio_four_extra/format/pls.pls'],
+    ['BBC Radio 5 Live', 'http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/http-icy-mp3-a/vpid/bbc_radio_five_live/format/pls.pls'],
+    ['BBC Radio 5 Live Sports Extra', 'http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/http-icy-mp3-a/vpid/bbc_radio_five_live_sports_extra/format/pls.pls'],
```
Comment (stefantalpalaru) [inline radio.py]: I can't open this from Italy. Must be geolocked.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
-bbc\_radio.py and rai\_radio.py are simple Internet radio scripts written in Python and Qt (using the [PyQt5][0] bindings). They were tested with python-2.7.14 and PyQt5-5.9 .
+## BBC & RAI Player
 
-Both use VLC by default but you can specify another player along with some arguments for it like this:
+radio.py is a simple Internet radio scripts written in Python and Qt (using the [PyQt5][0] bindings). They were tested with python-3.8.2 and PyQt5.14.2 .
```
Comment (stefantalpalaru) [inline README.md]: -> "It was tested..."

File: radio.py (hunk 1 above)
Comment (jacopomoioli) [reply radio.py]: Seems to be [techete'](https://www.raiplayradio.it/radiotechete/)
even rai is undecided, sometimes it is the "e" accented, sometimes it is the "e" followed by the apostrophe.

Comment (stefantalpalaru) [reply radio.py]: Trust me, I've been living in Italy for more than 13 years. What you're seeing there is the result of web developers not figuring out how to use Unicode, or not bothering to update old ASCII workarounds after moving the site to Unicode.

Comment (jacopomoioli) [reply radio.py]: yeah I know the italian style lol

**Comment by stefantalpalaru** [comment]
Thanks!

**Pull request merged**